packages/chem/quri_parts/chem/utils/excitations.py

```diff
@@ -0,0 +1,164 @@
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#      http://www.apache.org/licenses/LICENSE-2.0
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+from typing import Sequence
+
+from typing_extensions import TypeAlias
+
+from quri_parts.circuit import (
+    LinearMappedUnboundParametricQuantumCircuit,
+    LinearParameterFunction,
+    Parameter,
+    ParameterOrLinearFunction,
+)
+
+#: Represents the set of orbital indices involved in excitation. The first element is
+#: the index of an occupied orbital and the second element is the index of an
+#: unoccupied index.
+SingleExcitation: TypeAlias = tuple[int, int]
+
+#: Represents the set of orbital indices involved in excitation. The first and second
+#: element is the indices of occupied orbitals and the third and fourth element is the
+#: indices of unoccupied orbitals.
+DoubleExcitation: TypeAlias = tuple[int, int, int, int]
```

Comment:
In current documentation it is not shown that these are type aliases of tuple, so it may be better to mention it explicitly.

https://deploy-preview-40--quri-parts.netlify.app/quri_parts/chem/quri_parts.chem.utils.excitations.html